### ISSUE TITLE
Outputting != as '!' followed by '=' for C/C++, etc. code --not as ≠

### ISSUE THREAD

**Issue by preney** [gpoore/minted#209]
When using minted with C/C++ code all not equals, i.e., !=, are replaced with the single character '≠'. I cannot figure out how to make minted output '!' followed by '=' as this is what is in the code and that is what should appear in the output for languages such as C and C++. I cannot find any options in the documentation that control this and I've not found anything on the web answering this. (If it matters I am using XeLaTeX.) Thanks for any help you can provide! :-)

**Comment by gpoore** [comment]
This is not due to `minted`. Try
```latex
\documentclass{article}
\usepackage{minted}

\begin{document}

\begin{minted}{c}
a != b
\end{minted}

\end{document}
```
You are probably loading another package related to algorithms or code that redefines the behavior of the exclamation mark to cause this. I'd suggest taking the simple example above, and gradually adding your packages to that to see what is causing the issue. If you can narrow it down to one package, then you can check the documentation for that.

**Comment by gpoore** [comment]
Which font was it? I don't remember seeing anything like this before that was due to the font. I might want to add a note about this sort of thing to the FAQ.

**Comment by preney** [comment]
NOTE: XeLaTeX does not matter here. The font causes the ligatures to appear.

**Comment by gpoore** [comment]
Right. What I meant was that ligature issues would typically be encountered under XeLaTeX and LuaLaTeX, because it is easier to use other fonts with them.
